src/collective/collectionfilter/profiles/default/registry.xml

```diff
@@ -12,6 +12,17 @@
     </value>
   </record>
 
+  <record name="collective.collectionfilter.int_types">
+    <field type="plone.registry.field.Tuple">
+      <title>Target Collection Types</title>
+      <description>A list of fields that are int</description>
+      <value_type type="plone.registry.field.TextLine" />
+    </field>
+    <value>
+      <element>intfield</element>
```

Comment:
This looks like a project individual value. The default in the package is empty, so I would initialize it empty here too.